src/Team-Services-Bot.Api.UnitTests/Controllers/AuthorizeControllerTests.cs

```diff
@@ -44,7 +44,6 @@ public class AuthorizeControllerTests
 
             var controller = new AuthorizeController(
                 botService.Object,
-                new TelemetryClient(),
```

Comment:
There is "using Microsoft.ApplicationInsights;" which should be removed as well.